index.html

```diff
@@ -9,223 +9,263 @@
-
-                <button type="button" class="see-prj">See Project</button>
-              
-          
-        </div>
-               
-    </section>
-  </main>
-</body>
+        <!--third project-->
+
+        <article id="project3" class="project-container">
+          <!--background shapes-->
+          <div class="eclipse3"></div>
+
+          <figure class="project-image"></figure>
+
+          <h2 class="project-title">Multi-Post Stories</h2>
+          <p class="project-infor">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+
+          <ul class="used-tools">
+            <li class="project-list">css</li>
+
+            <li class="project-list">html</li>
+
+            <li class="project-list">bootstrap</li>
+
+            <li class="project-list">Ruby</li>
+          </ul>
+
+          <button type="button" class="see-prj">See Project</button>
+        </article>
+
+        <!--fourth project-->
+
+        <article id="project4" class="project-container">
+          <!--background shapes-->
+          <div class="eclipse4"></div>
+
+          <figure class="project-image"></figure>
+
+          <h2 class="project-title">Multi-Post Stories</h2>
+          <p class="project-infor">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text ever
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+
+          <ul class="used-tools">
+            <li class="project-list">css</li>
+
+            <li class="project-list">html</li>
+
+            <li class="project-list">bootstrap</li>
+
+            <li class="project-list">Ruby</li>
+          </ul>
+
+          <button type="button" class="see-prj">See Project</button>
+        </article>
+      </section>
+    </main>
+    <footer class="footer">
+      <div class="contact-us-format">
+        <h1 class="contact-us">CONTACT US</h1>
+
+        <p class="offer-help">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+
+     <form  action="https://formspree.io/f/xjvlllpk"  method="POST" class="contact" >
+        <input
+          type="text"
+          id="name"
+          name="name"
+          maxlength="30"
+          placeholder="Text input"
+        required/>
+
+        <input class="email" type="email" id="email" name="email" placeholder="jeremy@gmail.com" required/>
+
+        <textarea
+          class="message"
+          name="message"
+          id="message"
+          maxlength="500"
+          placeholder="Enter text here"
+          required></textarea>
```

Comment:
- [ ] as the previous reviewer asked you should use the right text color for your input as in the Figma design 
![image](https://user-images.githubusercontent.com/61790587/171290377-35cd4308-2bec-44dd-b1d3-594c3d20f191.png)